config/toml.go

```diff
@@ -550,7 +550,7 @@ func ResetTestRootWithChainID(testName string, chainID string) (*Config, error)
 	}
 
 	config := TestConfig().SetRoot(rootDir)
-	config.Instrumentation.Namespace = fmt.Sprintf("test-prom-%s-%s", testName, tmrand.Str(16))
+	config.Instrumentation.Namespace = fmt.Sprintf("%s_%s_%s", testName, chainID, tmrand.Str(16))
```

Comment:
At this point we might as well just use a UUID maybe? I don't think our tests actually do anything with the metrics.

Reply:
definitely.

Theoretically if it panics we'll be able to see this ID, and be able to know if the register call that caused this error is the one that we're currently running?